asyncapi_server.gemspec

```diff
@@ -19,15 +19,15 @@ Gem::Specification.new do |s|
 
   s.add_dependency "kaminari"
   s.add_dependency "api-pagination"
-  s.add_dependency "rails", "~> 4.0"
+  s.add_dependency "rails", ">= 4"
```

Comment:
`s.add_dependency "rails", "~> 5.0"` ?

Curious, wouldn't this mean Rails 4 apps would still be able to use this version, regardless of the following alterations?

Reply:
Yes, Rails 4 apps would still be compatible.